app/controllers/bulk_ingest_controller.rb

```diff
@@ -186,4 +165,10 @@ def workflow_states
     def workflow_class
       @workflow_class ||= DynamicChangeSet.new(resource_class.new).workflow_class
     end
+
+    def process_selected_files_params
+      return unless params[:selected_files]
+
+      params[:browse_everything] = params[:selected_files].fetch(:browse_everything, {})
```

Comment:
This is used later somewhere else? can you point out where to me?

Reply:
Sorry, yes, this is related to https://github.com/samvera/browse-everything/compare/master...jrgriffiniii:figgy-issues-2855-jrgriffiniii#diff-5805082baa0b62a6aa50acefa77dcf83R27 and https://github.com/samvera/browse-everything/compare/master...jrgriffiniii:figgy-issues-2855-jrgriffiniii#diff-5805082baa0b62a6aa50acefa77dcf83R12.

`browse-everything` assumes that the request parameter `browse_everything[:selected_files]` contains the values selected by users for upload, and this is indeed valid for the `BaseResourceController`.  However, the `BulkIngestController` assumes that the structure for the parameter is `selected_files[:browse_everything][:selected_files]`: https://github.com/pulibrary/figgy/blob/master/spec/controllers/bulk_ingest_controller_spec.rb#L123

I've just realized that I've forgotten to port this into `BaseResourceController`, I can adjust this and request a second review.